pkg/config/types.go

```diff
@@ -83,4 +83,8 @@ type Config struct {
 	// UserMappings is a list of mappings from AWS IAM User to
 	// Kubernetes username + groups.
 	UserMappings []UserMapping
+
+	// AWS Accounts that are allowed without an explicit user/role mapping
```

Comment:
We need to figure out what a good name for this is.  Names we have discussed internally include `AWSAccounts`, `MappedAccounts`, `AWSAccountsToMapToUser`, `AutoMappedAWSAccounts`. 

I open to suggestions on this.  I've been having trouble trying to find something that conveys the meaning and is concise at the same time.

Comment should start with name of struct member `MappedAccounts`

Reply:
I wasn't too sure about this either. Can change the name and comment when we decide on it.

Reply:
I think I like AutoMappedAWSAccounts.  It identifies what accounts we are talking about and what we are doing with them.  I would like either @mattmoyer or @nckturner to offer their input as well.  Naming things is hard.